<launch>

  <!-- Start the DirectedPerception PTU47 unit driver -->
  <node pkg="dp_ptu47_pan_tilt_stage" type="dp_ptu47" name="dp_ptu47" output="screen">
    <param name="port" value="/dev/ttyUSB3" />
    <param name="reset_on_start" value="0" />
  </node>

</launch>
